asabiris/handlers/kafkahandler.py

```
@@ -113,38 +114,52 @@ async def consume(self):
-			await self.handle_sms(msg)
-		elif msg_type == "push":
-			if self.App.SendPushOrchestrator is None:
-				L.warning("Push service is not configured. Discarding notification.")
+		# Set tenant context from Kafka message body (if present)
+		try:
+			if isinstance(msg, dict):
+				tenant = msg.get("tenant", None)
+			current_tenant = asab.contextvars.Tenant.get()
+			if tenant is not None and current_tenant is None:
+				token = asab.contextvars.Tenant.set(tenant)
+		except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)